### PR TITLE
Feat: create chart with its values.yaml

### LINKED ISSUE #27: Crear helm chart de la web
Para desplegar la web contenerizada https://hub.docker.com/r/cloudnativerioja/official-website se propone:

* Crear un chart de helm con un deployment/service/ingress

### PR DESCRIPTION
**Pull request by dcristobalhMad**
resolve #27 

* Ninja commit to add latest image to the repository
* Helm chart created

### CODE REVIEW

File: chart/official-website/Chart.yaml [hunk 1]
```diff
@@ -0,0 +1,24 @@
+apiVersion: v2
+name: official-website
+description: A Helm chart for Kubernetes
```
Comment (statickidz) [inline chart/official-website/Chart.yaml:3]: Add a more appropiate description here

File: chart/official-website/values.yaml [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+
+ingress:
+  enabled: true
+  className: ''
+  annotations:
+    cert-manager.io/cluster-issuer: 'letsencrypt-prod'
+    kubernetes.io/ingress.class: traefik
+    traefik.ingress.kubernetes.io/router.entrypoints: web,websecure
+  hosts:
+    - host: test.cluster.cloudnativerioja.com
```
Comment (statickidz) [inline chart/official-website/values.yaml:53]: should we use something more descriptive here?

**Review by statickidz — CHANGES_REQUESTED** [review]

**Comment by statickidz** [comment]
Huge work BTW!

File: chart/official-website/values.yaml (hunk 2 above)
Comment (dcristobalhMad) [reply chart/official-website/values.yaml:53]: Well, I was thinking in redirect this endpoint with a CNAME to the @ cloudnative record so it is not very important. But totally agree with you, better website.cluster.cloudnativerioja.com :rocket:

File: chart/official-website/Chart.yaml (hunk 1 above)
Comment (dcristobalhMad) [reply chart/official-website/Chart.yaml:3]: Yep sorry, I will do it

**Review by statickidz — APPROVED** [review]

**Pull request merged**